provisioning/bootstrap.yml

```diff
@@ -10,3 +10,49 @@
     - htpasswd-api
     - jeto
     - vagrant-worker
+
+  tasks:
+      - name: Update node to lastest version
+        shell:
+          executable=/bin/bash
+          npm install -g npm@latest
+
+      - name: Install node dependencies
+        shell:
+          executable=/bin/bash
+          chdir=/vagrant/projects/jeto/jeto/aurelia
+          npm install
+
+      - name: Install Gulp
+        shell:
+          executable=/bin/bash
+          npm install -g gulp
+
+      - name: Install jspm globally
+        shell:
+          executable=/bin/bash
+          npm install jspm -g
+
+      - name: Lock jspm
```

Comment:
Why is this needed in provisioning?
Running the command install jspm and put it in the `package.json` file (which is under version control). This is redundant with the `npm install` done above.